.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,781 @@
+    - When the pull-request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
+5. **Celebrate! 🎉**
+
+### Checklist: I have an enhancement 
+
+If you have an idea for Kirby that is not a brand new component or aimed at improving code quality; then you should submit an enhancement request!
+The process is the same as submitting a component request just using another issue-template: 
+
+1. **Open a [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the "enhancement request"-template** (See: "[The good: Issue](#issue)").
+2. **Follow steps 2 to 5 in "[Checklist: I have a component request](#checklist-i-have-a-component-request)".**
+
+### Checklist: I have found a bug 
+
+If you have found a bug in Kirby you should follow these steps: 
+
+1. **Report it by opening a [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the "bug report"-template** (See: "[The good: Issue](#issue)").
+2. **Get the bug verified** 
+    - To make sure the bug is not intended behaviour contact [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel) to have it verified; do not proceed until this is done. 
+3. **Implementation:** 
+    - Signal to others you are working on the issue by assigning yourself.
+    - Create a "fix"-branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master). 
+    - Create a test that reproduces the bug following guidelines in: "[The good: Test](#test)". 
+    - Write code to fix the bug
+4. **Follow steps 4 and 5 in "[Checklist: I have a component request](#checklist-i-have-a-component-request)".**
+
+## Setup environment and tooling for contribution
+
+<!-- The following section is responsible for teaching people how to get setup with 
+the repo such that they can get coding -->
+This section will help you get going with Kirby such that you can begin contributing. 
+
+### Clone and install the repository
+
+<!-- how to clone and get going with the kirby repo-->
+**Note:** make sure you have the right version of Node installed before installing the repository (see: [Node](#node)).
+
+1. Clone the repository: 
+
+``` 
+
+git clone https://github.com/kirbydesign/designsystem
+```
+
+2. Install dependencies: 
+
+``` 
+
+npm install
+```
+
+3. Start the cookbook server:
+
+``` 
+
+npm start
+```
+
+You should now be able to open `http://localhost:4200` in a browser to access the cookbook.
+
+### Node 
+
+<!-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. -->
+[Node](https://nodejs.org/en/) is the runtime used for development. 
+
+It is recommended to use `v12` for development as we currently do not support node LTS (see issue: [#1443](https://github.com/kirbydesign/designsystem/issues/1443)). 
```

Comment:
```suggestion
The supported version of NodeJS can be found in the file `.tool-versions` in the repository root.
```

Reply:
Can't seem to find `.tool-versions` did it end up in https://github.com/kirbydesign/designsystem/blob/master/.nvmrc instead?